python/keepsake/experiment.py

```diff
@@ -176,7 +176,7 @@ def to_json(self) -> Dict[str, Any]:
             "python_version": self.python_version,
             "python_packages": self.python_packages,
             "checkpoints": [c.to_json() for c in self.checkpoints],
-            "replicate_version": version,
+            "keepsake_version": version,
```

Comment:
Backwards compatibility issues here? (See go comments)

Reply:
I _think_ this will all go away by just setting `exp.KeepsakeVersion = exp.ReplicateVersion` above.